build.gradle

```diff
@@ -1,41 +1,196 @@
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.6.7'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.6.7'
+    compile 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.6.7'
+    compile 'javax.validation:validation-api:1.1.0.Final'
+
+    compile ('com.microsoft.azure:azure-storage:8.6.6') {
+        exclude group: 'org.slf4j', module: 'slf4j-api'
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
+        exclude group: 'com.google.guava', module: 'guava'
+        exclude group: 'org.slf4j', module: 'slf4j-api'
+        exclude group: 'org.apache.commons', module: 'commons-lang3'
+    }
+
+    testCompile "junit:junit:4.13"
+    testCompile "org.mockito:mockito-core:2.28.2"
+    testCompile "org.embulk:embulk-core:$embulkVersion:tests"
+    testCompile "org.embulk:embulk-deps:$embulkVersion"
+    testCompile "org.embulk:embulk-junit4:$embulkVersion"
+    testCompile "org.embulk:embulk-input-file:$embulkVersion"
+    testCompile "org.embulk:embulk-formatter-csv:$embulkVersion"
+    testCompile "org.embulk:embulk-parser-csv:$embulkVersion"
+}
+
+embulkPlugin {
+    mainClass = "org.embulk.output.azure_blob_storage.AzureBlobStorageFileOutputPlugin"
+    category = "output"
+    type = "azure_blob_storage"
+}
+
+javadoc {
+    options {
+        locale = "en_US"
+        encoding = "UTF-8"
+    }
+}
+
+jar {
+    from rootProject.file("LICENSE")
+}
+
+sourcesJar {
+    from rootProject.file("LICENSE")
+}
 
-    compile "com.microsoft.azure:azure-storage:8.6.6"
+javadocJar {
+    from rootProject.file("LICENSE")
+}
+
+// It should not publish a `.module` file in Maven Central.
+// https://docs.gradle.org/current/userguide/publishing_gradle_module_metadata.html#sub:disabling-gmm-publication
+tasks.withType(GenerateModuleMetadata) {
+    enabled = false
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = project.group
+            artifactId = project.name
+
+            from components.java
+
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                packaging "jar"
+
+                name = project.name
+                description = project.description
+                url = "https://www.embulk.org/"
+
+                developers {
+                    developer {
+                        name = "Satoshi Akama"
+                        email = "satoshiakama@gmail.com"
+                    }
+                    developer {
+                        name = "Hieu Duong"
+                        email = "duongminhhieu89@gmail.com"
+                    }
+                    developer {
+                        name = "Trung Huynh"
+                        email = "httrung90@gmail.com"
+                    }
+                    developer {
+                        name = "Tuan Bui"
+                        email = "xuantuan58@gmail.com"
+                    }
```

Comment:
Sorry if I have already asked for this before about another plugin. Is it okay to include your name and address (from Git log) like this, in `pom.xml` in Maven Central?